Makefile

```diff
@@ -18,13 +18,17 @@ submodule:
 
 .PHONY: compile
 compile:
+	cp contract/*.sol ./yui-ibc-solidity/contracts/
 	$(FORGE) build --config-path ./yui-ibc-solidity/foundry.toml
 
 .PHONY: abigen
 abigen: compile
-	@mkdir -p ./build/abi ./pkg/contract/ibchandler
-	@jq -r '.abi' ./yui-ibc-solidity/out/IBCHandler.sol/IBCHandler.json > ./build/abi/IBCHandler.abi
-	@$(ABIGEN) --abi ./build/abi/IBCHandler.abi --pkg ibchandler --out ./pkg/contract/ibchandler/ibchandler.go
+	@for a in IBCHandler Multicall3; do \
+	  b=$$(echo $$a | tr '[A-Z]' '[a-z]'); \
+	  mkdir -p ./build/abi ./pkg/contract/$$b; \
+	  jq -r '.abi' ./yui-ibc-solidity/out/$$a.sol/$$a.json > ./build/abi/$$a.abi; \
+	  $(ABIGEN) --abi ./build/abi/$$a.abi --pkg $$b --out ./pkg/contract/$$b/$$b.go; \
+	done
```

Comment:
In this code, is it assumed that yui-ibc-solidity contains `Multicall3`? I think it doesn't.

Reply:
At "compile" target of Makefile, copy Multicall3.sol to yui-ibc-solidity/
> cp contract/*.sol ./yui-ibc-solidity/contracts/

This yui-ibc-solidity directory is local copy, so the yui-ibc-solidity repository is not changed.

I've thought another way that run forge build on this ethereum-ibc-relay-chain directory, but it a bit toil work to keep same build option to yui-ibc-solidity's.